### ISSUE TITLE
Completely transparent libadwaita applications with libadwaita 1.6.0

### ISSUE THREAD

**Issue by Syphdias** [yshui/compton#1340]
### Platform
Arch Linux x86_64

### GPU, drivers, and screen setup
NVIDIA GeForce GTX 1070, propietary drivers 560.35.03, two monitors configured side-by-side with xrandr
```
name of display: :0
display: :0  screen: 0
direct rendering: Yes
Memory info (GL_NVX_gpu_memory_info):
    Dedicated video memory: 8192 MB
    Total available memory: 8192 MB
    Currently available dedicated video memory: 7056 MB
OpenGL vendor string: NVIDIA Corporation
OpenGL renderer string: NVIDIA GeForce GTX 1070/PCIe/SSE2
OpenGL core profile version string: 4.6.0 NVIDIA 560.35.03
OpenGL core profile shading language version string: 4.60 NVIDIA
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile

OpenGL version string: 4.6.0 NVIDIA 560.35.03
OpenGL shading language version string: 4.60 NVIDIA
OpenGL context flags: (none)
OpenGL profile mask: (none)

OpenGL ES profile version string: OpenGL ES 3.2 NVIDIA 560.35.03
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
```

### Environment
i3wm 4.23

### picom version
vgit-89c2c

<details>
<summary>Diagnostics</summary>

**Version:** vgit-89c2c

### Extensions:

* Shape: Yes
* RandR: Yes
* Present: Present

### Misc:

* Use Overlay: No
  (Another compositor is already running)
* Config file used: /home/syphdias/.config/picom.conf

### Drivers (inaccurate):

NVIDIA

### Backend: glx

* Driver vendors:
 * GLX: NVIDIA Corporation
 * GL: NVIDIA Corporation
* GL renderer: NVIDIA GeForce GTX 1070/PCIe/SSE2

### Backend: egl

* Driver vendors:
 * EGL: NVIDIA
 * GL: NVIDIA Corporation
* GL renderer: NVIDIA GeForce GTX 1070/PCIe/SSE2
</details>

### Configuration:
<details>
    <summary>Configuration file</summary>

```
❯ cat ~/.config/picom.conf
# Shadow
shadow = true;
# no-dnd-shadow = true;
no-dock-shadow = true;
shadow-radius = 7;
shadow-offset-x = 1;
shadow-offset-y = 1;
shadow-opacity = .3;
shadow-ignore-shaped = false;
shadow-exclude = [
        "name = 'Notification'",
# workaround for conky until it provides window properties:
        "override_redirect = 1 && !WM_CLASS@:s",
        "class_g ?= 'conky'",
        "class_g ?= 'Dmenu'",
# disable shadows for hidden windows:
        "_NET_WM_STATE@:32a *= '_NET_WM_STATE_HIDDEN'",
        "_GTK_FRAME_EXTENTS@:c",
# disables shadows on sticky windows:
#       "_NET_WM_STATE@:32a *= '_NET_WM_STATE_STICKY'",
# disables shadows on i3 frames
        "class_g ?= 'i3-frame'",
# fix gtk context menu
    "window_type = 'menu'",
    "window_type = 'dropdown_menu'",
    "window_type = 'popup_menu'",
    "window_type = 'tooltip'",
];

corner-radius = 4;
detect-rounded-corners = true;
rounded-corners-exclude = [
        "window_type = 'menu'",
        "window_type = 'dropdown_menu'",
        "window_type = 'popup_menu'",
        "window_type = 'utility'",
        "class_g = 'Polybar'",
        "class_g = 'Rofi'",
        "class_g = 'Dunst'"
];

blur: {
  method = "dual_kawase";
}
blur-background-exclude = [
#    "class_g = 'Dunst'"
   "name *= 'rect-overlay'", # fix screensharing
   "window_type = 'menu'", # fix blur of shadows of context menus (ff.)
   "window_type = 'dropdown_menu'",
   "window_type = 'popup_menu'",
   "window_type = 'tooltip'",
   "class_g ?= 'Peek'",
];

fading = false;
fade-delta = 1;
fade-in-step = 0.03;
fade-out-step = 0.03;
fade-exclude = [ ];

backend = "glx";
mark-wmwin-focused = true;
mark-ovredir-focused = true;
detect-client-opacity = true;
unredir-if-possible = true;
vsync = true;
dbe = false;
focus-exclude = [ "class_g = 'Cairo-clock'" ];
detect-transient = true;
detect-client-leader = true;
invert-color-include = [ ];
glx-copy-from-front = false;

wintypes :
{
  tooltip :
  {
    fade = true;
    shadow = false;
    opacity = 0.85;
    focus = true;
  };
  fullscreen :
  {
    fade = true;
    shadow = false;
    opacity = 1;
    focus = true;
  };
};
```
</details>

### Steps of reproduction
1. Install latest `libadwaita` (version 1.6.0)
2. (maybe also, but probably not: latest gtk4 (version 4.14.5))
3. Open adwaita-1-demo (from libadwaita-demos)

### Expected behavior
Normal Libadwaita app with bright/dark background 
![image](https://github.com/user-attachments/assets/54b788be-c286-4c63-844e-5fc128dbb6c8)

### Current Behavior
The entire app is made transparent
![image](https://github.com/user-attachments/assets/01166d83-5383-402e-9bb1-f862798456d8)

### Other details
I am not sure if that is a general problem with the new libadawaita or if I overselected in my config somewhere. I first noticed this in my terminal where the tabs got transparent.

**Comment by Syphdias** [comment]
![image](https://github.com/user-attachments/assets/8f69280d-3bd5-4a31-9238-27603894fbc9)

Terminal [ghostty](https://github.com/ghostty-org) (currently in closed beta) with opacity of 0.9. Works great for the terminal. But the UI should not be transparent or at least not _that_ transparent.

**Comment by Syphdias** [comment]
I was not on the latest gtk4 version (because I had other issues with it). This caused the issue.
